### PR TITLE
Traitor Item - Smuggler's Satchel

### PR DESCRIPTION
**Pull request by cheki-ss14**
## About the PR

Added the "Smuggler's Satchel" traitor item from SS13. This satchel can be purchased for 10 TC in the uplink, and comes in the "Smuggler" thief kit. It can be hidden under floor tiles. Crowbar a floor tile, place the satchel on the hull plate, anchor it with a wrench, then put the floor tile back! I'm sure this will only be used with good intentions.

Accidentally closed the other PR because of the merge conflicts with the great renaming blyat 

Link to closed PR discussion: https://github.com/Goob-Station/Goob-Station/pull/671

## Media

https://github.com/user-attachments/assets/d9b0b397-8f18-4434-8ab3-6467c73c40be

## Requirements

- [X ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X ] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Added Smuggler's Satchel as an uplink and thief item.

## Summary by CodeRabbit

- **New Features**
	- Introduced the "Smuggler's Satchel," a new item that can be concealed under floor tiles, enhancing stealth gameplay.
	- Added the "Clothing Satchel Smuggler" to the thief's toolbox sets, expanding available items for players.
	- Expanded the uplink catalog with the new item available for purchase using Telecrystals.

- **Bug Fixes**
	- Updated descriptions for better clarity regarding item contents in the backpack.

- **Documentation**
	- Enhanced item listings and descriptions in the catalog for improved user understanding.

### CODE REVIEW

File: Resources/Prototypes/Catalog/thief_toolbox_sets.yml [hunk 1]
```diff
@@ -107,3 +107,4 @@
   - SmokeGrenade
   - SmokeGrenade
   - SmokeGrenade
+  - ClothingSatchelSmuggler
```
Comment (Piras314) [inline Resources/Prototypes/Catalog/thief_toolbox_sets.yml:110]: add # Goobstation to the end of this line

**Review by Piras314 — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/_Goobstation/store/uplink-catalog.ftl [hunk 2]
```diff
@@ -19,3 +19,6 @@ uplink-mobroach-microbomb-desc = Call in a handy mothroach equipped with a micro
 
 uplink-advanced-combat-medipen-name = Advanced Combat Medipen
 uplink-advanced-combat-medipen-desc = An advanced single-use medipen containing chemicals that will rapidly heal most types of damage taken in a fight.
+
+uplink-smugglers-satchel-name = Smuggler's Satchel
+uplink-smugglers-satchel-desc = A rugged looking satchel that can be hidden under floor tiles. Simply remove a floor tile, anchor the satchel to the plating, then put the tile back!
```
Comment (Piras314) [inline Resources/Locale/en-US/_Goobstation/store/uplink-catalog.ftl:24]: just call it a rugged satchel not a rugged-looking satchel

**Review by Piras314 — APPROVED** [review]

**Comment by cheki-ss14** [comment]
@Piras314  Wanted to make sure that you've seen the previous discussion about the TC cost, as well as your thoughts on it and any balance changed you'd like me to make.

**Comment by Piras314** [comment]
I've seen it, I think either way is fine tbh.

**Review by whateverusername0 — APPROVED** [review]

**Comment by ElPidor** [comment]
Wouldnt it be a little better if it gets renamed to just satchels, so secoff wont stop me if im having one?

**Comment by Piras314** [comment]
@ElPidor is right, make its ftl name just Satchel or whatever regular satchels are called and make it 15tc

**Comment by Piras314** [comment]
@cheki-ss14

**Comment by cheki-ss14** [comment]
@Piras314 Done

**Review by Piras314 — APPROVED** [review]

**Pull request merged**